Apsitvarkom.DataAccess/Apsitvarkom.DataAccess.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="11.0.1" />
+    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
+  </ItemGroup>
```

Comment:
Both of these are probably not needed in `Apsitvarkom.DataAccess` project. Rule of thumb is to only include packages where you use it (I don't think we're using it in this project).

Reply:
Discussed this below. Will be sorted out.

> Yes, that is possible. I have excessively left the package in the DataAccess project too, as I wasn't sure whether we should generate already mapped results or not, so I will revert it.